src/ServiceManagement/Services/Commands.Utilities/Websites/WebsitesClient.cs

```diff
@@ -1360,9 +1360,43 @@ private DeploymentBaseOptions CreateRemoteDeploymentBaseOptions(string websiteNa
                 AuthenticationType = "Basic",
                 TempAgent = false
             };
+            
+            var azureSessionUserAgent = GetUserAgentForAzureSession();
```

Comment:
Can you point me to the location of tests for PublishWebProject which is the public function here. I was not able to find any. Otherwise I will have to make GetDeploymentBaseOptionsUserAgent() public which doesn't seem to be right